ci-operator/step-registry/cucushift/installer/rehearse/gcp/cases/valid-confidential-computing/provision/cucushift-installer-rehearse-gcp-cases-valid-confidential-computing-provision-commands.sh

```diff
@@ -0,0 +1,344 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# -----------------------------------------
+# OCP-60212 - [IPI-on-GCP] Install with invalid settings of Confidential Computing on GCP	
+# -----------------------------------------
+
+# save the exit code for junit xml file generated in step gather-must-gather
+# pre configuration steps before running installation, exit code 100 if failed,
+# save to install-pre-config-status.txt
+# post check steps after cluster installation, exit code 101 if failed,
+# save to install-post-check-status.txt
+EXIT_CODE=101
+trap 'if [[ "$?" == 0 ]]; then EXIT_CODE=0; fi; echo "${EXIT_CODE}" > "${SHARED_DIR}/install-post-check-status.txt"; post_actions' EXIT TERM
```

Comment:
From the pj-rehearse job [log](https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/63439/rehearse-63439-periodic-ci-openshift-verification-tests-main-installation-nightly-4.19-gcp-ipi-valid-confidential-computing-f14/1934447725735055360):
```
/bin/bash: line 1: post_actions: command not found
```